docs/site/src/components/ApiMenu.js

```diff
@@ -0,0 +1,68 @@
+// @flow weak
+
+import React, { PureComponent, PropTypes } from 'react';
+import { camelCase } from 'docs/site/src/utils/helpers';
```

Comment:
Awesome, let's follow that convention. I think that we have an import eslint enforcing some part of it. Like having related import at the bottom.